frontend/src/styles.css

```diff
@@ -9,6 +9,9 @@
     html, body  {
         margin: none;
         padding: none;
+        font-family: "Inter Tight", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+            Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
+            "Segoe UI Symbol";
     }
     h1 {
```

Comment:
Again, we should have almost nothing in here apart from the html tag and body tag styling and the imports.

So ideally all the styles here are moved to where they are used or we use primeNG to simplify it.